lms/views/lti/deep_linking.py

```diff
@@ -76,6 +76,7 @@ def deep_linking_launch(context, request):
             "lti_message_type": "ContentItemSelection",
             "lti_version": request.parsed_params["lti_version"],
         },
+        prompt_title=request.product.plugin.misc.deep_linking_prompt_title,
```

Comment:
The field name sounds to me like it is a string rather than a boolean ("the title for the prompt" rather than "should prompt for a title"). Possible alternatives: `prompt_for_title`, `show_assignment_title_input`, `show_assignment_title_prompt`.